ElectricityBillApi/Models/ElectricProviders.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+
+namespace ElectricityBillApi.Models
+{
+    public class ElectricProviders : List<ElectricProvider>
```

Comment:
Avoid inheriting base data structures. You're not extending the capabilities of a list, you are implementing business requirements for electric providers. List is an implementation detail. Therefore, instead add your own interface and hide the fact that the underlying type is a list.

Reply:
You mean in that case I should rather go with something like a `Providers` class with a field `List<ElectricProvider>` ?